src/utils/timeutils.ts

```diff
@@ -14,39 +11,25 @@ function formatSecondsToHours(seconds: number) {
 }
 
 function timeAgo(lastSeen: Date) {
-  moment.locale("nb");
-  const lastSeenTime = moment(lastSeen).add(TIMEZONE_ADJUSTMENT_OFFSET_MS);
+  const lastSeenTime = moment.tz(lastSeen, "Europe/Oslo");
 
   return lastSeenTime.fromNow();
 }
 
 export const calculateSessionTime = (
   sessionDuration: number, // Current duration calculated by the backend
-  currentTime: Moment,
   lastSeen?: Date,
 ) => {
   if (!lastSeen) return "";
 
-  // Might want to remove this if raspberry pi and presence happen to both have the same timezone
-  const nowTime = currentTime; // Current time
-
-  // Parse the start time using Moment.js
-  const lastSeenDate = moment(lastSeen).add(TIMEZONE_ADJUSTMENT_OFFSET_MS); //Adjust for wrong timezone
-
-  // Calculate the difference in seconds
-  const updatedDuration = sessionDuration + nowTime.diff(lastSeenDate) / 1000;
```

Comment:
Is it intentional to remove this part? It will fix the issue of the extra hour, as it will not try to add time between fetching the number from the backend. But after deleting this we will no longer get updated times every second.

I'm all for not increasing the second all the time - and we refetch from the backend frequently enough, so go ahead.

If you do delete these lines, you should also delete the useEffect that triggers this to be run every second, as that would not have any use and just cause unnecessary rendering.